clang/lib/DPCT/ASTTraversal.cpp

```diff
@@ -911,6 +913,8 @@ void IncludesCallbacks::InclusionDirective(
         DpctGlobalInfo::getHelperFilesCustomizationLevel() ==
             HelperFilesCustomizationLevel::HFCL_All) {
       DpctGlobalInfo::getInstance().insertHeader(HashLoc, HT_MKL_RNG);
+      requestFeature(HelperFeatureEnum::RngUtils_non_local_include_dependency,
```

Comment:
if custom level is All,  do we need call extra requestFeature to include header file?